modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/ckeditor5/BaseEditor.tsx

```diff
@@ -30,12 +32,20 @@ const BaseEditor = ({
 	data?: string;
 	disabled?: boolean;
 	editor: any;
+	enableHiddenInput?: boolean;
+	name?: string;
```

Comment:
To make code a bit clearer, can we call these
- `hiddenInputEnabled`
- `hiddenInputName`

Reply:
Ah, that's a good point. I'll rename.